### ISSUE TITLE
Build Install

### ISSUE THREAD

**Issue by robweber** [Blythie/BlythieBuildInstaller#1]
Sorry for the issues with communications - weird the forum won't let you send a PM. In any case, describe what you have going on and I'll read through it here.

**Comment by Blythie** [comment]
No worries mate got there in the end:) What is it i have a build installer and what it does it looks at and installer text file then determines what build to pull down.

The plugin will then install the zip backup (build) but the problem i got is that it doesn't install on a fresh installer, it just overwrites it

So im trying to get the addon to install the build but do a clean wipe a the begin. The plugin can be found on this repo :)

Hope that makes sense :)

**Comment by robweber** [comment]
From what I can tell it looks like you are trying to download a zip to the special://home/addons/packages folder, and then extract from that zip to the special://home/addons directory. Is this right? If so I can see that maybe being a problem. I can't imagine the OS likes something like that very much. Have you tried using like the "special://temp" directory for the download? 

I guess another question is are you trying to overwrite the addons in the folder with ones from your build? IE wipe that folder and replace them all, or just add some? Wiping them might be a problem as they may be in use by Kodi somewhere and the file can't be deleted. Do you get any log information in Kodi when you try to do this?

**Comment by Blythie** [comment]
Yeah that right, and no i haven't tried that yet do you think it will work?

Yeah so what i want to happen addon to run the installer the installer
clean kodi to a clean build with nothing on and then the zip overwrites
everything.

I thats a point and no i didn't get any.

Do you think its possible it the addon is created in a different way?

**Comment by robweber** [comment]
I would try using the temp directory just to see what happens. I think adding addons isn't a problem but deleting the ones there while Kodi is running may prove a problem. 

The code here looks very similar to something I was asked to look at a while back. Something called the MTP Media Box wizard. Is this a modification of that code? At one point there was a site that had Kodi "themes" which were essentially pre-built skins/addon customizations. They had some type of deployment wizard as well. Can't remember the name of the site but their downloader may have other features, worth googling around for.

**Comment by Blythie** [comment]
Do you think we can just get the main wizard add-on to call up the addon linked below?

https://github.com/Blythie/BlythieBuildInstaller/blob/master/plugin.video.elmorewizard.zip
